### ISSUE TITLE
Give a hint about search hits on deprecated concepts

### ISSUE THREAD

**Issue by osma** [NatLibFi/Skosmos#537]
## At which URL did you encounter the problem?
http://finto.fi/yso/fi/search?clang=fi&q=abstrakti&vocabs=

## What steps will reproduce the problem?
1. Search YSO for "abstrakti" in Finnish

## What is the expected output? What do you see instead?

Expected to get some kind of hint that there is a deprecated concept with that label within YSO. Instead getting only other results.

Need to discuss how to implement this. Some options:
1. Display the deprecated concepts in the search result list, but clearly distinguish them visually (e.g. gray color)
2. Display a link at the top of search results list, "show 1 matching deprecated concept"

Also the autocomplete list should be adjusted accordingly, probably the deprecated concepts should be somehow visible there as well.

**Comment by osma** [comment]
Possibly better example: "bluesfestivaalit" (which was deprecated recently in YSO)

Could be implemented as part of advanced search. But then the user would not get the hint that the concept used to exist, unless he was specifically looking for it.

**Comment by kouralex** [comment]
This would be rather fascinating feature. I think both options presented are good and could possibly be combined. What do you think of having a cookie for "showing the matched deprecated concept(s)"?

**Comment by osma** [comment]
I don't think we want more widgets in the UI. This could go to the advanced search form (#411)
